spec/dor_config.rb

```diff
@@ -22,11 +22,7 @@
     catalog.url 'http://example.edu/catalog/mods'
   end
 
-  gsearch do
-    rest_url 'https://example.edu/gsearch/rest'
-    url      'https://example.edu/solr/gsearch'
-  end
-  solrizer.url     'https://example.edu/solr/solrizer'
+  solr.url     'https://example.edu/solr/solrizer'
```

Comment:
same comment as the other example.  i mean not a big deal if you'd prefer not, but more consistent w/ real solr urls too.